packages/api/src/command/medical/admin/upload-doc.ts

```diff
@@ -32,34 +34,42 @@ export async function createAndUploadDocReference({
   };
 }): Promise<DocumentReference> {
   const patient = await getPatientOrFail({ id: patientId, cxId });
-  const organization = await getOrganizationOrFail({ cxId });
 
   const fhirApi = makeFhirApi(cxId);
-
-  const now = dayjs();
+  const refDate = dayjs();
+  const orgId = smallId();
+  const orgRef = `org${orgId}`;
+  const practitionerId = smallId();
+  const practitionerRef = `org${practitionerId}`;
 
   const metriportContent = createDocReferenceContent({
     contentType: file.mimetype,
     size: file.size,
-    creation: now.format(),
+    creation: refDate.format(),
     fileName: file.originalname,
     location: `${docContributionUrl}?fileName=${file.originalname}`,
     extension: [metriportDataSourceExtension],
+    format: "urn:ihe:pcc:xphr:2007",
   });
 
   // WHEN OPENING THIS FOR CX'S NEED TO UPDATE THE CONTENT
-  const data: DocumentReference = {
-    resourceType: "DocumentReference",
+  const data: DocumentReference = getFHIRDocRef(patient.id, {
     id: docId,
     contained: [
       {
         resourceType: "Organization",
-        id: organization.id,
-        name: organization.data.name,
+        id: orgRef,
+        name: `Hospital ${orgRef}`,
       },
       {
-        resourceType: "Patient",
-        id: patient.id,
+        resourceType: "Practitioner",
+        id: practitionerRef,
+        name: [
+          {
+            family: `Last ${practitionerId}`,
+            given: [`First ${practitionerId}`],
```

Comment:
don't understand why we have Last/First in the name - isn't this implicit based on the family/given property?

Reply:
can ignore above, just realized this is a func for testing